internal/pkg/stream/ecs.go

```diff
@@ -43,7 +44,14 @@ func (d ECSDeployment) isPrimary() bool {
 }
 
 func (d ECSDeployment) done() bool {
-	return d.RolloutState != rollOutInProgress
+	switch d.RolloutState {
+	case rollOutFailed:
+		return true
+	case rollOutCompleted:
+		return d.DesiredCount == d.RunningCount
```

Comment:
Is it possible that the deployment is `COMPLETED` but the tasks are not all running?

Reply:
Yeah T_T when you change the manifest for example and set `count: 3` from `count: 2`. Then the rollout is completed  from the beginning, but the `desiredCount=3`, `pendingCount=1` and `runningCount=2`